plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSplitSource.java

```diff
@@ -104,6 +103,7 @@ private HiveSplitSource(
             PerBucket queues,
             int maxInitialSplits,
             DataSize maxOutstandingSplitsSize,
+            double minimumAssignedSplitWeight,
```

Comment:
If we decide to remove the session property, you could just pass a `HiveSplitWeightProvider` directly instead of the `double` value here instead.